Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365_test.py

```diff
@@ -533,6 +533,37 @@ def test_parse_incident_from_item():
     assert incident['attachment']
 
 
+@pytest.mark.parametrize("mime_content", [
+    "Hello, this is a sample email with non-ASCII characters: é, ñ, ü.",
+    "Hello, this is a sample email with ASCII characters",
+])
+def test_parse_incident_from_item_non_ascii(mime_content):
+    """
+    Given:
+        - Message item with attachment that contains non-ASCII characters or only ASCII characters.
+
+    When:
+        - Parsing incident from item
+
+    Verify:
+        - Parsing runs successfully
```

Comment:
Can we check in the test that the parsing ran successfully and the output is what we want?